README.md

```diff
@@ -35,12 +35,38 @@ Email: info@iliosproject.org
 
 # Development of Ilios
 
-You can build a run a local version of Ilios easily using [docker](https://www.docker.com/).
-There are convenient clients for Mac and Windows and instructions Linux.
+The Ilios API is a PHP application which required a MySql server.  The easiest way to get a working
```

Comment:
The Ilios API is a PHP application which requires a MySQL server.  The easiest way to get a working